<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LumberLog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="pteLog">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClear">
    <property name="title">
     <string>Actio&amp;ns</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionIgnore_linefeeds"/>
    <addaction name="actionTimestamps"/>
   </widget>
   <addaction name="menuClear"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionClear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionIgnore_linefeeds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Ignore linefeeds</string>
   </property>
  </action>
  <action name="actionTimestamps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Timestamps</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
